src/gui.rs

```diff
@@ -71,8 +72,8 @@ impl NWBView {
                     }
                     if is_open {
                         let mut test_plot = Box::new(super::plot::ContextMenus::default());
-                        set_open(&mut self.open_windows, &group.handler.name(), is_open);
                         test_plot.show(ctx, &mut is_open, group);
+                        set_open(&mut self.open_windows, &group.handler.name(), is_open);
```

Comment:
Ahahaha the trick was just to swap them :D

Reply:
yeah :laughing: